cometd-java/cometd-java-benchmark/cometd-java-benchmark-client/src/main/java/org/cometd/benchmark/client/CometDLoadClient.java

```diff
@@ -741,7 +751,7 @@ private Histogram printResults(long messageCount, long expectedCount, long elaps
             );
         }
 
-        AtomicHistogram histogram = this.histogram.copy();
+        Histogram histogram = allHistograms.stream().reduce(new Histogram(TimeUnit.MICROSECONDS.toNanos(1), TimeUnit.MINUTES.toNanos(1), 3), (h1, h2)-> {h1.add(h2); return h1;});
```

Comment:
Split the lambda into multiple lines and braces at EOL.

Reply:
Ok.